src/main/java/com/internet/shop/model/ShoppingCart.java

```diff
@@ -0,0 +1,10 @@
+package com.internet.shop.model;
+
+import java.util.List;
+
+public class ShoppingCart {
+    private Long id;
+    private Long userId;
+    private List<Product> products;
+
```

Comment:
let's create constructor, getters and setters

note that we won't receive products from outside in the constructor